### PR TITLE
Added contributing guidelines file

### PR DESCRIPTION
**Pull request by Deepu178**
I have added a contributing guidelines file that how can a newbie contribute to the Niceplaces and made few changes to the readme file to add links to the contributing file.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Contributing to Niceplaces
```
Comment (lorenzovngl) [inline contributing.md]: Better to use the name of the repository: Nice Places Website

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
```
Comment (lorenzovngl) [inline contributing.md]: Wrong link: the correct one is https://docs.github.com/en/get-started/quickstart/github-flow

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+3. If you've changed APIs, update the documentation.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+## Any contributions you make will be under the MIT Software License
+In short, when you submit code changes, your submissions are understood to be under the same [MIT License](http://choosealicense.com/licenses/mit/) that covers the project. Feel free to contact the maintainers if that's a concern.
+
+## Report bugs using Github's [issues](https://github.com/niceplaces/website/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](); it's that easy!
```
Comment (lorenzovngl) [inline contributing.md]: Missing link URL, you can use https://github.com/niceplaces/website/issues

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+- Steps to reproduce
+  - Be specific!
+  - Give sample code if you can. [stackoverflow question](http://stackoverflow.com/q/12488905/180626) includes sample code that *anyone* with a base R setup can run to reproduce what I was seeing
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+People *love* thorough bug reports. I'm not even kidding.
+
+## Use a Consistent Coding Style
```
Comment (lorenzovngl) [inline contributing.md]: Better to remove this section, the project don't have a unified style

File: contributing.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+I'm again borrowing these from [Facebook's Guidelines](https://github.com/facebook/draft-js/blob/a9316a723f9e918afde44dea68b5f9f39b7d9b00/CONTRIBUTING.md)
+
+* 2 spaces for indentation rather than tabs
+* You can try running `npm run lint` for style unification
+
+## License
+By contributing, you agree that your contributions will be licensed under its MIT License.
+
+## References
+This document was adapted from the open-source contribution guidelines for [Facebook's Draft](https://github.com/facebook/draft-js/blob/a9316a723f9e918afde44dea68b5f9f39b7d9b00/CONTRIBUTING.md)
```
Comment (lorenzovngl) [inline contributing.md]: The link is broken, however I think is good to refer to https://gist.github.com/briandk/3d2e8b3ec8daf5a27a62.
So something like this:

This document was adapted from [this open-source contribution guidelines](https://gist.github.com/briandk/3d2e8b3ec8daf5a27a62).

File: README.md [hunk 6]
```diff
@@ -40,3 +40,7 @@ The source code available in this repository is licensed under the [MIT license]
 5. Open `http://localhost/niceplaces-website/` in your browser.
 
 *TODO: add the instructions to configure the local MySQL database and a Google Maps API key*
+
+Do you want to contribute to Niceplaces?
```
Comment (lorenzovngl) [inline README.md]: Better to use the name of the repository: Nice Places Website

**Review by lorenzovngl — CHANGES_REQUESTED** [review]
I added some changes I would like to see. Also, I would like to name the file of contribution guidelines CONTRIBUTING.md and not contributing.md.

Thank you

File: contributing.md (hunk 1 above)
Comment (Deepu178) [reply contributing.md]: Changed to Nice Places Website

File: contributing.md (hunk 2 above)
Comment (Deepu178) [reply contributing.md]: Replaced with the above link

File: contributing.md (hunk 3 above)
Comment (Deepu178) [reply contributing.md]: Used the above URL.

File: contributing.md (hunk 4 above)
Comment (Deepu178) [reply contributing.md]: Removed the  ## Use a consistent Coding Style 
section

File: contributing.md (hunk 5 above)
Comment (Deepu178) [reply contributing.md]: Replaced with the above link: https://gist.github.com/briandk/3d2e8b3ec8daf5a27a62

File: README.md (hunk 6 above)
Comment (Deepu178) [reply README.md]: Use the name of the repository : Nice Places Website

**Comment by Deepu178** [comment]
I made changes as you suggested and kindly look into it.
Thanks

**Review by lorenzovngl — APPROVED** [review]

**Pull request merged**